### PR TITLE
change delete topic - return delete elements and their status

### PR DESCRIPTION
**Pull request by damian0o**

### CODE REVIEW

File: src/components/organisms/DeleteTopic.js [hunk 1]
```diff
@@ -18,23 +22,29 @@ const DeleteTopic = ({selectedElement, toggleSelection, destination, isModalVisi
...
+          `);
+        };
+        if(failedMeetings.length > 0){
+          toast.error(`
+            Nie udało się usunąć następujących tematów:
+            ${failedMeetings.toString()}
+          `);
+        };
+      })
+      // .catch((err) => {
```
Comment (damian0o) [inline src/components/organisms/DeleteTopic.js:42]: Removed unused code.

File: src/actions/topics.js [hunk 2]
```diff
@@ -2,6 +2,10 @@ export const GET_TOPICS_REQUEST = 'GET_TOPICS_REQUEST';
 export const GET_TOPICS_RECEIVED = 'GET_TOPICS_RECEIVED';
 export const GET_TOPICS_ERROR = 'GET_TOPICS_ERROR';
 
+export const SET_SUCCESSFULLY_REMOVED_MEETINGS = 'SET_SUCCESSFULLY_REMOVED_MEETINGS';
```
Comment (damian0o) [inline src/actions/topics.js:5]: We should stick to some covention when naming events - all events from group should share same prefix

REMOVE_MEETINGS_SUCESS
REMOVE_MEETINGS_CLEAR
REMOVE_MEETINGS_ERROR

File: src/actions/topics.js [hunk 3]
```diff
@@ -2,6 +2,10 @@ export const GET_TOPICS_REQUEST = 'GET_TOPICS_REQUEST';
...
+export const CLEAR_SUCCESSFULLY_REMOVED_MEETINGS = 'CLEAR_SUCCESSFULLY_REMOVED_MEETINGS';
+export const REMOVE_TOPICS_ERROR = 'REMOVE_TOPICS_ERROR';
```
Comment (damian0o) [inline src/actions/topics.js:7]: Should we change topics to meetings ?

File: src/selectors/fetchTopics.js [hunk 4]
```diff
@@ -1,4 +1,7 @@
-import {getTopicsPending, getTopicsSuccess, getTopicsError} from 'actions/topics';
+import {
+  getTopicsPending, getTopicsSuccess, getTopicsError,
+  setSuccessfullyRemovedMeetings, removeTopicsError
```
Comment (damian0o) [inline src/selectors/fetchTopics.js:3]: Among all of this topic related functions one Meetings looks strage.
setSuccessfullyRemovedMeetings -> i believe it should be change to setSuccessfullyRemovedTopics

File: src/selectors/fetchTopics.js [hunk 5]
```diff
@@ -34,16 +37,33 @@ export const addTopic = (dataToSend) => {
...
+          method: 'DELETE',
+          body: JSON.stringify(element)
+        });
+        if(response.ok) return ({success: true, meetingTopic: element.topic});
+        else return ({success: false, errorText: await response.text()});
     };
 
     try {
-      await Promise.all(removeElements(selectedElements, destination));
+      const response = await Promise.all(selectedElements.map(element => removeElement(element)));
```
Comment (damian0o) [inline src/selectors/fetchTopics.js:50]: I dont know what processing here . Do we process meetings or topics?

**Comment by pawelwojtkiewicz** [comment]
> We should name things accordingly to functionality they provide - if we are handling topics all functions should have topics in name (not meetings) -
> 
> or at least we should consider to rename those functions (if both topics and meetings are handled similarly) without topics or meetings inside
> 
> TOAST_SUCCESS_MESSAGE
> TOAST_ERROR_MESSAGE
> TOAST_CLEAR_MESSAGE
> or something more specific

Could those functions be renamed as:
DELETE_TOPICS_SET_REMOVED_TOPICS - set successfully removed topics 
DELETE_TOPICS_SET_NOT_REMOVED_TOPICS - set unsuccessfully removed topics
DELETE_TOPICS_ ERROR - generally error with remove topic/topics

File: src/actions/topics.js (hunk 2 above)
Comment (pawelwojtkiewicz) [reply src/actions/topics.js:5]: Could those functions be renamed as:
DELETE_TOPICS_SET_REMOVED_TOPICS - set successfully removed topics
DELETE_TOPICS_SET_NOT_REMOVED_TOPICS - set unsuccessfully removed topics
DELETE_TOPICS_ ERROR - generally error with remove topic/topics

File: src/actions/topics.js (hunk 3 above)
Comment (pawelwojtkiewicz) [reply src/actions/topics.js:7]: No, we should stay in topics if we working with topics

File: src/selectors/fetchTopics.js (hunk 4 above)
Comment (pawelwojtkiewicz) [reply src/selectors/fetchTopics.js:3]: Yes, of course. My bad here. In the application we have a little mess with concepts "meetings", "topics".

File: src/actions/topics.js (hunk 3 above)
Comment (damian0o) [reply src/actions/topics.js]: So we should rename 2 entries above meetings to topics

**Review by pawelwojtkiewicz — COMMENTED** [review]
Good

**Review by pawelwojtkiewicz — APPROVED** [review]
Good

**Pull request merged**